src/frequenz/client/dispatch/__main__.py

```diff
@@ -32,24 +32,22 @@
 )
 from ._client import Client
 
-DEFAULT_DISPATCH_API_HOST = "fz-0004.frequenz.io"
-DEFAULT_DISPATCH_API_PORT = 50051
+DEFAULT_DISPATCH_API_URL = "grpc://fz-0004.frequenz.io:50051?ssl=true"
```

Comment:
Not sure if this is based on code that was upgraded to client-base 0.6.0, but if that's the case now `ssl=true` is the default.